JavaScript/JavaScript.sublime-syntax

```diff
@@ -207,13 +207,13 @@ contexts:
         8: storage.type.function.arrow.js
     - match: |-
         (?x)
-        (\b_?[A-Z][$\w]*)?
+        (\b_?[a-zA-Z_?.$]*)?
```

Comment:
Oh I just copied this regex from the one for non arrow functions. I think it is actually designed this way so that you can do things like.

```javascript
SomeThing.SomeOtherThing.AnotherThing.whatever = () => {}
```

Not the prettiest, but I am not sure if sublime syntaxes support repeating patterns like that.

Technically it should probably use the same pattern as the one on line 211 to actually match the correct characters.